src/generalizedlinearmixedmodel.jl

```diff
@@ -284,16 +307,24 @@ function fit!(
     end
     setpar! = fast ? setθ! : setβθ!
     prog = ProgressUnknown("Minimizing"; showspeed=true)
+    # start from zero for the initial call to obj before optimization
+    iter = 0
     function obj(x, g)
         isempty(g) || throw(ArgumentError("g should be empty for this objective"))
         val = deviance(pirls!(setpar!(m, x), fast, verbose), nAGQ)
+        !isnothing(fitlog) && iszero(rem(iter, thin)) && push!(fitlog, (copy(x), val))
```

Comment:
```suggestion
        !isnothing(fitlog) && isone(rem(iter, thin)) && push!(fitlog, (copy(x), val))
```
I know I suggested `iszero` but it may be better to ensure that the record starts with the first evaluation.

Reply:
The initial evaluation at theta_0 is handled explicitly below -- or did you want to capture the first step after that?

Reply:
No I didn't see that the initial evaluation is explicitly handled.  It should be okay to leave this as `iszero` then.